A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/Com/A320_Neo_Com.js

```diff
@@ -44,16 +44,24 @@ class A320_Neo_Com extends BaseAirliners {
     }
 
     onInteractionEvent(_args) {
+        const eventString = String(_args);
+        const searchString = "DCDU_BTN_";
+        const index = eventString.indexOf(searchString);
+        if (index !== -1) {
+            const event = eventString.substring(index + searchString.length);
+            console.log("DCDU event captured: " + event);
```

Comment:
Remove `console.log` for performance

Reply:
It's only called when you actually press a button. Shouldn't have an impact on performance